test/test_prototype_builtin_datasets.py

```diff
@@ -52,6 +52,8 @@ def test_sample(self, test_home, dataset_mock, config):
 
         try:
             sample = next(iter(dataset))
+        except StopIteration:
+            raise AssertionError("Unable to draw any sample.") from None
```

Comment:
It makes this case more expressive. Before you got an error message like `Drawing a sample resulted in the error above` while "above" is a plain `StopIteration`. If you are familiar with datapipes, you should be able to interpret this, but new contributors might not be. I've fixed this here, because we were hitting this exact case.